### PR TITLE
[MB-11988] Create a History & Audit Logging Guide

### PR DESCRIPTION
**Pull request by kctruss**
While ample documentation exists around the individual parts of the H&AL feature, it seemed pertinent to collect a reference guide as an easier entry point for future devs.

### CODE REVIEW

File: docs/backend/guides/guide-to-history-and-audit-logging.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# History and Audit Logging Guide
+
+## Tracking Event History from End to End
+- [Database Triggers](#database-triggers)
+- [AppContext](#appcontext-in-context)
+- [Fetcher](#move-history-fetcher)
+- [Frontend](#frontend-event-templates)
+
+### Database Triggers
+Move history events are created by [Event Triggers](docs/backend/guides/how-to/add-an-event-trigger/) and are collected in a single :lock:[move history table](https://dp3.atlassian.net/wiki/spaces/MT/pages/1634992192/PO9+Technical+Implementation) which captures what event took place, what data changed, and which user initiated the event.
```
Comment (michaelvaldes-truss) [inline docs/backend/guides/guide-to-history-and-audit-logging.md:14]: **suggestion**: These internal links weren't working on my machine, I think they might need to start with a slash

```suggestion
Move history events are created by [Event Triggers](/docs/backend/guides/how-to/add-an-event-trigger/) and are collected in a single :lock:[move history table](https://dp3.atlassian.net/wiki/spaces/MT/pages/1634992192/PO9+Technical+Implementation) which captures what event took place, what data changed, and which user initiated the event.
```

**Review by michaelvaldes-truss — CHANGES_REQUESTED** [review]
This is a great overview and starting point for all of our move history information!
You could also include images for our detail type designs if you wanted, but I don't think it's necessary.

File: docs/backend/guides/guide-to-history-and-audit-logging.md (hunk 1 above)
Comment (kctruss) [reply docs/backend/guides/guide-to-history-and-audit-logging.md]: Ahhhh thanks for catching that. I pushed a fix.

**Review by michaelvaldes-truss — APPROVED** [review]
lgtm!

**Pull request merged**